packages/transport-webrtc/src/private-to-private/signaling-stream-handler.ts

```diff
@@ -93,11 +93,10 @@ export async function handleIncomingStream ({ peerConnection, stream, signal, co
   } catch (err: any) {
     if (getConnectionState(peerConnection) !== 'connected') {
       log.error('error while handling signaling stream from peer %a', connection.remoteAddr, err)
-
       peerConnection.close()
       throw err
     } else {
-      log('error while handling signaling stream from peer %a, ignoring as the RTCPeerConnection is already connected', connection.remoteAddr, err)
+      log.error('error while handling signaling stream from peer %a', connection.remoteAddr, err)
```

Comment:
Please reinstate the previous log message and log level.  The error token is missing as well.